readme.md

```diff
@@ -38,64 +38,63 @@ $ npm -v
 
 * **Question:** What is the difference between forking and cloning a repository as opposed to just cloning a repository?
 
-* **Your Answer:**
+* **Your Answer:** When you Fork the repo then you are adding it under your repository.  Just cloning would two versions of the same repositoty.  If there
 
 ---
 
 - [ ] Run `npm init -y` from the command line
 
 * **Question:** What does `npm init` do? How does the `-y` flag modify that command?
 
-* **Your Answer:**
+* **Your Answer:** NPM initializes the program.  The -y means "Yes," it just assumes yes and moves on.
```

Comment:
This is a bit incorrect. The `npm init` command creates a `package.json` file -- that's it. It does create it with some sensible defaults. The `-y` flag does indeed say "yes" to all the questions you might be asked when initializing your file.